bin/build-ubuntu.sh

```diff
@@ -63,7 +79,7 @@ PATH="${DIR_THIRDPARTY}/bde-tools/bin:$PATH"
 
 if [ ! -e "${DIR_BUILD}/bde/.complete" ]; then
     pushd "${DIR_THIRDPARTY}/bde"
-    eval "$(bbs_build_env -u opt_64_cpp17 -b "${DIR_BUILD}/bde" -i "${DIR_INSTALL}")"
+    eval "$(bbs_build_env -u opt_64_cpp17_pic -b "${DIR_BUILD}/bde" -i "${DIR_INSTALL}")"
```

Comment:
You are right, it's not needed. I added "position-independent code" flag to be able to link BDE to the plugin. But we don't need it, since BDE is linked to the broker and resolved by the plugin in runtime. So, I'll remove it

Reply:
d